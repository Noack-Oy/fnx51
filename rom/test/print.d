print.lst print.hex: print.asm ../global/variables.equ ../serial/sfr.equ ../global/init.inc ../serial/init.inc ../serial/rx.inc ../serial/tx.inc ../print/char.inc ../print/text.inc ../print/hex.inc ../print/int.inc ../util/regbank.inc ../util/panic.inc
