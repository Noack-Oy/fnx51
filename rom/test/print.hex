:1000000075814090013A8582328583333130900189
:100010001E314790012B3147E4315D240120D60782
:1000200090012E314780F290012B314770EB314720
:10003000780079007B0C3172E82455F8E93400F936
:100040004012DB097B0C90012B314780E990012E97
:10005000314780E290012B3147314790012BE4F882
:10006000F9FAFB7C01FDFEFF317D3147E82CF8E910
:100070003DF9EA3EFAEB3FFBECC8FCEDC9FDEECAE8
:10008000FEEFCBFF50E2317D31473147E431A1240F
:100090000120D60790012E314780F290012B314785
:1000A00070EB31477480318E240120D21120D607A5
:1000B00090012E314780EF90012B314780E890016D
:1000C0002B31473147780079007B0A31EDE8243342
:1000D000F8E93400F94012DB097B0A90012B314723
:1000E00080E990012E314780E290012B3147314762
:1000F000780079807B0A31D2E8247FF8E93400F96E
:1001000020D212DB097B0A90012B314780E8900155
:100110002E314780E190012B31473147211C486542
:100120006C6C6F2C20776F726C64210D0A002000BC
:10013000759852759ABF759B1E223099FDC299F52C
:100140009922C032C03322C0E0E8C0E0E4F89360F6
:1001500006314208E870F7D0E0F8D0E022C43161FF
:10016000C4C0E0540F2490D43440D444203142D051
:10017000E022C0E0E9315DE8315DD0E022C0E0EB93
:10018000315DEA315DE9315DE8315DD0E022C0E00A
:10019000C0F030E710F5F0742D3142E5F0F4048042
:1001A00004C0E0C0F088F0C0F0780075F0648460AE
:1001B000052430314208E5F075F00A84286005987E
:1001C00024303142E5F024303142D0E0F8D0F0D094
:1001D000E022C0E0E8C0E0E9C0E030E718742D316B
:1001E00042E8F42401F8E9F43400F98008C0E0E8BA
:1001F000C0E0E9C0E0EBC0E0C0F07A0075F0FFC3FA
:10020000E89410F8E99427F905F050F4E82410F880
:10021000E93427F9E5F06005243031420A75F0FF32
:10022000C3E894E8F8E99403F905F050F4E824E809
:10023000F8E93403F9E5F02A60069A243031420ADD
:1002400075F0FFC3E89464F8E99400F905F050F400
:10025000E82464F8E93400F9E5F02A60069A2430CD
:1002600031420AE875F00A842A60059A2430314246
:10027000E5F024303142D0F0D0E0FAD0E0F9D0E01F
:04028000F8D0E022B0
:00000001FF
