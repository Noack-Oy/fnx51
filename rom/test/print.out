BOOTLOADER
> U
< U
> :020000050B00EE
< :020000050B00EE7B.\r\n
WRITE: print.hex
> :1000000075814090013A8582328583333130900189
< :1000000075814090013A8582328583333130900189.\r\n
> :100010001E314790012B3147E4315D240120D60782
< :100010001E314790012B3147E4315D240120D60782.\r\n
> :1000200090012E314780F290012B314770EB314720
< :1000200090012E314780F290012B314770EB314720.\r\n
> :10003000780079007B0C3172E82455F8E93400F936
< :10003000780079007B0C3172E82455F8E93400F936.\r\n
> :100040004012DB097B0C90012B314780E990012E97
< :100040004012DB097B0C90012B314780E990012E97.\r\n
> :10005000314780E290012B3147314790012BE4F882
< :10005000314780E290012B3147314790012BE4F882.\r\n
> :10006000F9FAFB7C01FDFEFF317D3147E82CF8E910
< :10006000F9FAFB7C01FDFEFF317D3147E82CF8E910.\r\n
> :100070003DF9EA3EFAEB3FFBECC8FCEDC9FDEECAE8
< :100070003DF9EA3EFAEB3FFBECC8FCEDC9FDEECAE8.\r\n
> :10008000FEEFCBFF50E2317D31473147E431A1240F
< :10008000FEEFCBFF50E2317D31473147E431A1240F.\r\n
> :100090000120D60790012E314780F290012B314785
< :100090000120D60790012E314780F290012B314785.\r\n
> :1000A00070EB31477480318E240120D21120D607A5
< :1000A00070EB31477480318E240120D21120D607A5.\r\n
> :1000B00090012E314780EF90012B314780E890016D
< :1000B00090012E314780EF90012B314780E890016D.\r\n
> :1000C0002B31473147780079007B0A31EDE8243342
< :1000C0002B31473147780079007B0A31EDE8243342.\r\n
> :1000D000F8E93400F94012DB097B0A90012B314723
< :1000D000F8E93400F94012DB097B0A90012B314723.\r\n
> :1000E00080E990012E314780E290012B3147314762
< :1000E00080E990012E314780E290012B3147314762.\r\n
> :1000F000780079807B0A31D2E8247FF8E93400F96E
< :1000F000780079807B0A31D2E8247FF8E93400F96E.\r\n
> :1001000020D212DB097B0A90012B314780E8900155
< :1001000020D212DB097B0A90012B314780E8900155.\r\n
> :100110002E314780E190012B31473147211C486542
< :100110002E314780E190012B31473147211C486542.\r\n
> :100120006C6C6F2C20776F726C64210D0A002000BC
< :100120006C6C6F2C20776F726C64210D0A002000BC.\r\n
> :10013000759852759ABF759B1E223099FDC299F52C
< :10013000759852759ABF759B1E223099FDC299F52C.\r\n
> :100140009922C032C03322C0E0E8C0E0E4F89360F6
< :100140009922C032C03322C0E0E8C0E0E4F89360F6.\r\n
> :1001500006314208E870F7D0E0F8D0E022C43161FF
< :1001500006314208E870F7D0E0F8D0E022C43161FF.\r\n
> :10016000C4C0E0540F2490D43440D444203142D051
< :10016000C4C0E0540F2490D43440D444203142D051.\r\n
> :10017000E022C0E0E9315DE8315DD0E022C0E0EB93
< :10017000E022C0E0E9315DE8315DD0E022C0E0EB93.\r\n
> :10018000315DEA315DE9315DE8315DD0E022C0E00A
< :10018000315DEA315DE9315DE8315DD0E022C0E00A.\r\n
> :10019000C0F030E710F5F0742D3142E5F0F4048042
< :10019000C0F030E710F5F0742D3142E5F0F4048042.\r\n
> :1001A00004C0E0C0F088F0C0F0780075F0648460AE
< :1001A00004C0E0C0F088F0C0F0780075F0648460AE.\r\n
> :1001B000052430314208E5F075F00A84286005987E
< :1001B000052430314208E5F075F00A84286005987E.\r\n
> :1001C00024303142E5F024303142D0E0F8D0F0D094
< :1001C00024303142E5F024303142D0E0F8D0F0D094.\r\n
> :1001D000E022C0E0E8C0E0E9C0E030E718742D316B
< :1001D000E022C0E0E8C0E0E9C0E030E718742D316B.\r\n
> :1001E00042E8F42401F8E9F43400F98008C0E0E8BA
< :1001E00042E8F42401F8E9F43400F98008C0E0E8BA.\r\n
> :1001F000C0E0E9C0E0EBC0E0C0F07A0075F0FFC3FA
< :1001F000C0E0E9C0E0EBC0E0C0F07A0075F0FFC3FA.\r\n
> :10020000E89410F8E99427F905F050F4E82410F880
< :10020000E89410F8E99427F905F050F4E82410F880.\r\n
> :10021000E93427F9E5F06005243031420A75F0FF32
< :10021000E93427F9E5F06005243031420A75F0FF32.\r\n
> :10022000C3E894E8F8E99403F905F050F4E824E809
< :10022000C3E894E8F8E99403F905F050F4E824E809.\r\n
> :10023000F8E93403F9E5F02A60069A243031420ADD
< :10023000F8E93403F9E5F02A60069A243031420ADD.\r\n
> :1002400075F0FFC3E89464F8E99400F905F050F400
< :1002400075F0FFC3E89464F8E99400F905F050F400.\r\n
> :10025000E82464F8E93400F9E5F02A60069A2430CD
< :10025000E82464F8E93400F9E5F02A60069A2430CD.\r\n
> :1002600031420AE875F00A842A60059A2430314246
< :1002600031420AE875F00A842A60059A2430314246.\r\n
> :10027000E5F024303142D0F0D0E0FAD0E0F9D0E01F
< :10027000E5F024303142D0F0D0E0FAD0E0F9D0E01F.\r\n
> :04028000F8D0E022B0
< :04028000F8D0E022B0.\r\n
> :00000001FF
< :00000001FF.\r\n
RESET
Hello, world!

00 01 02 03 04 05 06 07 08 09 0a 0b 0c 0d 0e 0f
10 11 12 13 14 15 16 17 18 19 1a 1b 1c 1d 1e 1f
20 21 22 23 24 25 26 27 28 29 2a 2b 2c 2d 2e 2f
30 31 32 33 34 35 36 37 38 39 3a 3b 3c 3d 3e 3f
40 41 42 43 44 45 46 47 48 49 4a 4b 4c 4d 4e 4f
50 51 52 53 54 55 56 57 58 59 5a 5b 5c 5d 5e 5f
60 61 62 63 64 65 66 67 68 69 6a 6b 6c 6d 6e 6f
70 71 72 73 74 75 76 77 78 79 7a 7b 7c 7d 7e 7f
80 81 82 83 84 85 86 87 88 89 8a 8b 8c 8d 8e 8f
90 91 92 93 94 95 96 97 98 99 9a 9b 9c 9d 9e 9f
a0 a1 a2 a3 a4 a5 a6 a7 a8 a9 aa ab ac ad ae af
b0 b1 b2 b3 b4 b5 b6 b7 b8 b9 ba bb bc bd be bf
c0 c1 c2 c3 c4 c5 c6 c7 c8 c9 ca cb cc cd ce cf
d0 d1 d2 d3 d4 d5 d6 d7 d8 d9 da db dc dd de df
e0 e1 e2 e3 e4 e5 e6 e7 e8 e9 ea eb ec ed ee ef
f0 f1 f2 f3 f4 f5 f6 f7 f8 f9 fa fb fc fd fe ff

0000 0055 00aa 00ff 0154 01a9 01fe 0253 02a8 02fd 0352 03a7
03fc 0451 04a6 04fb 0550 05a5 05fa 064f 06a4 06f9 074e 07a3
07f8 084d 08a2 08f7 094c 09a1 09f6 0a4b 0aa0 0af5 0b4a 0b9f
0bf4 0c49 0c9e 0cf3 0d48 0d9d 0df2 0e47 0e9c 0ef1 0f46 0f9b
0ff0 1045 109a 10ef 1144 1199 11ee 1243 1298 12ed 1342 1397
13ec 1441 1496 14eb 1540 1595 15ea 163f 1694 16e9 173e 1793
17e8 183d 1892 18e7 193c 1991 19e6 1a3b 1a90 1ae5 1b3a 1b8f
1be4 1c39 1c8e 1ce3 1d38 1d8d 1de2 1e37 1e8c 1ee1 1f36 1f8b
1fe0 2035 208a 20df 2134 2189 21de 2233 2288 22dd 2332 2387
23dc 2431 2486 24db 2530 2585 25da 262f 2684 26d9 272e 2783
27d8 282d 2882 28d7 292c 2981 29d6 2a2b 2a80 2ad5 2b2a 2b7f
2bd4 2c29 2c7e 2cd3 2d28 2d7d 2dd2 2e27 2e7c 2ed1 2f26 2f7b
2fd0 3025 307a 30cf 3124 3179 31ce 3223 3278 32cd 3322 3377
33cc 3421 3476 34cb 3520 3575 35ca 361f 3674 36c9 371e 3773
37c8 381d 3872 38c7 391c 3971 39c6 3a1b 3a70 3ac5 3b1a 3b6f
3bc4 3c19 3c6e 3cc3 3d18 3d6d 3dc2 3e17 3e6c 3ec1 3f16 3f6b
3fc0 4015 406a 40bf 4114 4169 41be 4213 4268 42bd 4312 4367
43bc 4411 4466 44bb 4510 4565 45ba 460f 4664 46b9 470e 4763
47b8 480d 4862 48b7 490c 4961 49b6 4a0b 4a60 4ab5 4b0a 4b5f
4bb4 4c09 4c5e 4cb3 4d08 4d5d 4db2 4e07 4e5c 4eb1 4f06 4f5b
4fb0 5005 505a 50af 5104 5159 51ae 5203 5258 52ad 5302 5357
53ac 5401 5456 54ab 5500 5555 55aa 55ff 5654 56a9 56fe 5753
57a8 57fd 5852 58a7 58fc 5951 59a6 59fb 5a50 5aa5 5afa 5b4f
5ba4 5bf9 5c4e 5ca3 5cf8 5d4d 5da2 5df7 5e4c 5ea1 5ef6 5f4b
5fa0 5ff5 604a 609f 60f4 6149 619e 61f3 6248 629d 62f2 6347
639c 63f1 6446 649b 64f0 6545 659a 65ef 6644 6699 66ee 6743
6798 67ed 6842 6897 68ec 6941 6996 69eb 6a40 6a95 6aea 6b3f
6b94 6be9 6c3e 6c93 6ce8 6d3d 6d92 6de7 6e3c 6e91 6ee6 6f3b
6f90 6fe5 703a 708f 70e4 7139 718e 71e3 7238 728d 72e2 7337
738c 73e1 7436 748b 74e0 7535 758a 75df 7634 7689 76de 7733
7788 77dd 7832 7887 78dc 7931 7986 79db 7a30 7a85 7ada 7b2f
7b84 7bd9 7c2e 7c83 7cd8 7d2d 7d82 7dd7 7e2c 7e81 7ed6 7f2b
7f80 7fd5 802a 807f 80d4 8129 817e 81d3 8228 827d 82d2 8327
837c 83d1 8426 847b 84d0 8525 857a 85cf 8624 8679 86ce 8723
8778 87cd 8822 8877 88cc 8921 8976 89cb 8a20 8a75 8aca 8b1f
8b74 8bc9 8c1e 8c73 8cc8 8d1d 8d72 8dc7 8e1c 8e71 8ec6 8f1b
8f70 8fc5 901a 906f 90c4 9119 916e 91c3 9218 926d 92c2 9317
936c 93c1 9416 946b 94c0 9515 956a 95bf 9614 9669 96be 9713
9768 97bd 9812 9867 98bc 9911 9966 99bb 9a10 9a65 9aba 9b0f
9b64 9bb9 9c0e 9c63 9cb8 9d0d 9d62 9db7 9e0c 9e61 9eb6 9f0b
9f60 9fb5 a00a a05f a0b4 a109 a15e a1b3 a208 a25d a2b2 a307
a35c a3b1 a406 a45b a4b0 a505 a55a a5af a604 a659 a6ae a703
a758 a7ad a802 a857 a8ac a901 a956 a9ab aa00 aa55 aaaa aaff
ab54 aba9 abfe ac53 aca8 acfd ad52 ada7 adfc ae51 aea6 aefb
af50 afa5 affa b04f b0a4 b0f9 b14e b1a3 b1f8 b24d b2a2 b2f7
b34c b3a1 b3f6 b44b b4a0 b4f5 b54a b59f b5f4 b649 b69e b6f3
b748 b79d b7f2 b847 b89c b8f1 b946 b99b b9f0 ba45 ba9a baef
bb44 bb99 bbee bc43 bc98 bced bd42 bd97 bdec be41 be96 beeb
bf40 bf95 bfea c03f c094 c0e9 c13e c193 c1e8 c23d c292 c2e7
c33c c391 c3e6 c43b c490 c4e5 c53a c58f c5e4 c639 c68e c6e3
c738 c78d c7e2 c837 c88c c8e1 c936 c98b c9e0 ca35 ca8a cadf
cb34 cb89 cbde cc33 cc88 ccdd cd32 cd87 cddc ce31 ce86 cedb
cf30 cf85 cfda d02f d084 d0d9 d12e d183 d1d8 d22d d282 d2d7
d32c d381 d3d6 d42b d480 d4d5 d52a d57f d5d4 d629 d67e d6d3
d728 d77d d7d2 d827 d87c d8d1 d926 d97b d9d0 da25 da7a dacf
db24 db79 dbce dc23 dc78 dccd dd22 dd77 ddcc de21 de76 decb
df20 df75 dfca e01f e074 e0c9 e11e e173 e1c8 e21d e272 e2c7
e31c e371 e3c6 e41b e470 e4c5 e51a e56f e5c4 e619 e66e e6c3
e718 e76d e7c2 e817 e86c e8c1 e916 e96b e9c0 ea15 ea6a eabf
eb14 eb69 ebbe ec13 ec68 ecbd ed12 ed67 edbc ee11 ee66 eebb
ef10 ef65 efba f00f f064 f0b9 f10e f163 f1b8 f20d f262 f2b7
f30c f361 f3b6 f40b f460 f4b5 f50a f55f f5b4 f609 f65e f6b3
f708 f75d f7b2 f807 f85c f8b1 f906 f95b f9b0 fa05 fa5a faaf
fb04 fb59 fbae fc03 fc58 fcad fd02 fd57 fdac fe01 fe56 feab
ff00 ff55 ffaa ffff

00000000
00000001
00000001
00000002
00000003
00000005
00000008
0000000d
00000015
00000022
00000037
00000059
00000090
000000e9
00000179
00000262
000003db
0000063d
00000a18
00001055
00001a6d
00002ac2
0000452f
00006ff1
0000b520
00012511
0001da31
0002ff42
0004d973
0007d8b5
000cb228
00148add
00213d05
0035c7e2
005704e7
008cccc9
00e3d1b0
01709e79
02547029
03c50ea2
06197ecb
09de8d6d
0ff80c38
19d699a5
29cea5dd
43a53f82
6d73e55f
b11924e1

0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15
16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31
32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47
48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63
64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79
80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95
96 97 98 99 100 101 102 103 104 105 106 107 108 109 110 111
112 113 114 115 116 117 118 119 120 121 122 123 124 125 126 127
128 129 130 131 132 133 134 135 136 137 138 139 140 141 142 143
144 145 146 147 148 149 150 151 152 153 154 155 156 157 158 159
160 161 162 163 164 165 166 167 168 169 170 171 172 173 174 175
176 177 178 179 180 181 182 183 184 185 186 187 188 189 190 191
192 193 194 195 196 197 198 199 200 201 202 203 204 205 206 207
208 209 210 211 212 213 214 215 216 217 218 219 220 221 222 223
224 225 226 227 228 229 230 231 232 233 234 235 236 237 238 239
240 241 242 243 244 245 246 247 248 249 250 251 252 253 254 255

-128 -127 -126 -125 -124 -123 -122 -121 -120 -119 -118 -117 -116 -115 -114 -113
-112 -111 -110 -109 -108 -107 -106 -105 -104 -103 -102 -101 -100 -99 -98 -97
-96 -95 -94 -93 -92 -91 -90 -89 -88 -87 -86 -85 -84 -83 -82 -81
-80 -79 -78 -77 -76 -75 -74 -73 -72 -71 -70 -69 -68 -67 -66 -65
-64 -63 -62 -61 -60 -59 -58 -57 -56 -55 -54 -53 -52 -51 -50 -49
-48 -47 -46 -45 -44 -43 -42 -41 -40 -39 -38 -37 -36 -35 -34 -33
-32 -31 -30 -29 -28 -27 -26 -25 -24 -23 -22 -21 -20 -19 -18 -17
-16 -15 -14 -13 -12 -11 -10 -9 -8 -7 -6 -5 -4 -3 -2 -1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15
16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31
32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47
48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63
64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79
80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95
96 97 98 99 100 101 102 103 104 105 106 107 108 109 110 111
112 113 114 115 116 117 118 119 120 121 122 123 124 125 126 127

0 51 102 153 204 255 306 357 408 459
510 561 612 663 714 765 816 867 918 969
1020 1071 1122 1173 1224 1275 1326 1377 1428 1479
1530 1581 1632 1683 1734 1785 1836 1887 1938 1989
2040 2091 2142 2193 2244 2295 2346 2397 2448 2499
2550 2601 2652 2703 2754 2805 2856 2907 2958 3009
3060 3111 3162 3213 3264 3315 3366 3417 3468 3519
3570 3621 3672 3723 3774 3825 3876 3927 3978 4029
4080 4131 4182 4233 4284 4335 4386 4437 4488 4539
4590 4641 4692 4743 4794 4845 4896 4947 4998 5049
5100 5151 5202 5253 5304 5355 5406 5457 5508 5559
5610 5661 5712 5763 5814 5865 5916 5967 6018 6069
6120 6171 6222 6273 6324 6375 6426 6477 6528 6579
6630 6681 6732 6783 6834 6885 6936 6987 7038 7089
7140 7191 7242 7293 7344 7395 7446 7497 7548 7599
7650 7701 7752 7803 7854 7905 7956 8007 8058 8109
8160 8211 8262 8313 8364 8415 8466 8517 8568 8619
8670 8721 8772 8823 8874 8925 8976 9027 9078 9129
9180 9231 9282 9333 9384 9435 9486 9537 9588 9639
9690 9741 9792 9843 9894 9945 9996 10047 10098 10149
10200 10251 10302 10353 10404 10455 10506 10557 10608 10659
10710 10761 10812 10863 10914 10965 11016 11067 11118 11169
11220 11271 11322 11373 11424 11475 11526 11577 11628 11679
11730 11781 11832 11883 11934 11985 12036 12087 12138 12189
12240 12291 12342 12393 12444 12495 12546 12597 12648 12699
12750 12801 12852 12903 12954 13005 13056 13107 13158 13209
13260 13311 13362 13413 13464 13515 13566 13617 13668 13719
13770 13821 13872 13923 13974 14025 14076 14127 14178 14229
14280 14331 14382 14433 14484 14535 14586 14637 14688 14739
14790 14841 14892 14943 14994 15045 15096 15147 15198 15249
15300 15351 15402 15453 15504 15555 15606 15657 15708 15759
15810 15861 15912 15963 16014 16065 16116 16167 16218 16269
16320 16371 16422 16473 16524 16575 16626 16677 16728 16779
16830 16881 16932 16983 17034 17085 17136 17187 17238 17289
17340 17391 17442 17493 17544 17595 17646 17697 17748 17799
17850 17901 17952 18003 18054 18105 18156 18207 18258 18309
18360 18411 18462 18513 18564 18615 18666 18717 18768 18819
18870 18921 18972 19023 19074 19125 19176 19227 19278 19329
19380 19431 19482 19533 19584 19635 19686 19737 19788 19839
19890 19941 19992 20043 20094 20145 20196 20247 20298 20349
20400 20451 20502 20553 20604 20655 20706 20757 20808 20859
20910 20961 21012 21063 21114 21165 21216 21267 21318 21369
21420 21471 21522 21573 21624 21675 21726 21777 21828 21879
21930 21981 22032 22083 22134 22185 22236 22287 22338 22389
22440 22491 22542 22593 22644 22695 22746 22797 22848 22899
22950 23001 23052 23103 23154 23205 23256 23307 23358 23409
23460 23511 23562 23613 23664 23715 23766 23817 23868 23919
23970 24021 24072 24123 24174 24225 24276 24327 24378 24429
24480 24531 24582 24633 24684 24735 24786 24837 24888 24939
24990 25041 25092 25143 25194 25245 25296 25347 25398 25449
25500 25551 25602 25653 25704 25755 25806 25857 25908 25959
26010 26061 26112 26163 26214 26265 26316 26367 26418 26469
26520 26571 26622 26673 26724 26775 26826 26877 26928 26979
27030 27081 27132 27183 27234 27285 27336 27387 27438 27489
27540 27591 27642 27693 27744 27795 27846 27897 27948 27999
28050 28101 28152 28203 28254 28305 28356 28407 28458 28509
28560 28611 28662 28713 28764 28815 28866 28917 28968 29019
29070 29121 29172 29223 29274 29325 29376 29427 29478 29529
29580 29631 29682 29733 29784 29835 29886 29937 29988 30039
30090 30141 30192 30243 30294 30345 30396 30447 30498 30549
30600 30651 30702 30753 30804 30855 30906 30957 31008 31059
31110 31161 31212 31263 31314 31365 31416 31467 31518 31569
31620 31671 31722 31773 31824 31875 31926 31977 32028 32079
32130 32181 32232 32283 32334 32385 32436 32487 32538 32589
32640 32691 32742 32793 32844 32895 32946 32997 33048 33099
33150 33201 33252 33303 33354 33405 33456 33507 33558 33609
33660 33711 33762 33813 33864 33915 33966 34017 34068 34119
34170 34221 34272 34323 34374 34425 34476 34527 34578 34629
34680 34731 34782 34833 34884 34935 34986 35037 35088 35139
35190 35241 35292 35343 35394 35445 35496 35547 35598 35649
35700 35751 35802 35853 35904 35955 36006 36057 36108 36159
36210 36261 36312 36363 36414 36465 36516 36567 36618 36669
36720 36771 36822 36873 36924 36975 37026 37077 37128 37179
37230 37281 37332 37383 37434 37485 37536 37587 37638 37689
37740 37791 37842 37893 37944 37995 38046 38097 38148 38199
38250 38301 38352 38403 38454 38505 38556 38607 38658 38709
38760 38811 38862 38913 38964 39015 39066 39117 39168 39219
39270 39321 39372 39423 39474 39525 39576 39627 39678 39729
39780 39831 39882 39933 39984 40035 40086 40137 40188 40239
40290 40341 40392 40443 40494 40545 40596 40647 40698 40749
40800 40851 40902 40953 41004 41055 41106 41157 41208 41259
41310 41361 41412 41463 41514 41565 41616 41667 41718 41769
41820 41871 41922 41973 42024 42075 42126 42177 42228 42279
42330 42381 42432 42483 42534 42585 42636 42687 42738 42789
42840 42891 42942 42993 43044 43095 43146 43197 43248 43299
43350 43401 43452 43503 43554 43605 43656 43707 43758 43809
43860 43911 43962 44013 44064 44115 44166 44217 44268 44319
44370 44421 44472 44523 44574 44625 44676 44727 44778 44829
44880 44931 44982 45033 45084 45135 45186 45237 45288 45339
45390 45441 45492 45543 45594 45645 45696 45747 45798 45849
45900 45951 46002 46053 46104 46155 46206 46257 46308 46359
46410 46461 46512 46563 46614 46665 46716 46767 46818 46869
46920 46971 47022 47073 47124 47175 47226 47277 47328 47379
47430 47481 47532 47583 47634 47685 47736 47787 47838 47889
47940 47991 48042 48093 48144 48195 48246 48297 48348 48399
48450 48501 48552 48603 48654 48705 48756 48807 48858 48909
48960 49011 49062 49113 49164 49215 49266 49317 49368 49419
49470 49521 49572 49623 49674 49725 49776 49827 49878 49929
49980 50031 50082 50133 50184 50235 50286 50337 50388 50439
50490 50541 50592 50643 50694 50745 50796 50847 50898 50949
51000 51051 51102 51153 51204 51255 51306 51357 51408 51459
51510 51561 51612 51663 51714 51765 51816 51867 51918 51969
52020 52071 52122 52173 52224 52275 52326 52377 52428 52479
52530 52581 52632 52683 52734 52785 52836 52887 52938 52989
53040 53091 53142 53193 53244 53295 53346 53397 53448 53499
53550 53601 53652 53703 53754 53805 53856 53907 53958 54009
54060 54111 54162 54213 54264 54315 54366 54417 54468 54519
54570 54621 54672 54723 54774 54825 54876 54927 54978 55029
55080 55131 55182 55233 55284 55335 55386 55437 55488 55539
55590 55641 55692 55743 55794 55845 55896 55947 55998 56049
56100 56151 56202 56253 56304 56355 56406 56457 56508 56559
56610 56661 56712 56763 56814 56865 56916 56967 57018 57069
57120 57171 57222 57273 57324 57375 57426 57477 57528 57579
57630 57681 57732 57783 57834 57885 57936 57987 58038 58089
58140 58191 58242 58293 58344 58395 58446 58497 58548 58599
58650 58701 58752 58803 58854 58905 58956 59007 59058 59109
59160 59211 59262 59313 59364 59415 59466 59517 59568 59619
59670 59721 59772 59823 59874 59925 59976 60027 60078 60129
60180 60231 60282 60333 60384 60435 60486 60537 60588 60639
60690 60741 60792 60843 60894 60945 60996 61047 61098 61149
61200 61251 61302 61353 61404 61455 61506 61557 61608 61659
61710 61761 61812 61863 61914 61965 62016 62067 62118 62169
62220 62271 62322 62373 62424 62475 62526 62577 62628 62679
62730 62781 62832 62883 62934 62985 63036 63087 63138 63189
63240 63291 63342 63393 63444 63495 63546 63597 63648 63699
63750 63801 63852 63903 63954 64005 64056 64107 64158 64209
64260 64311 64362 64413 64464 64515 64566 64617 64668 64719
64770 64821 64872 64923 64974 65025 65076 65127 65178 65229
65280 65331 65382 65433 65484 65535

-32768 -32641 -32514 -32387 -32260 -32133 -32006 -31879 -31752 -31625
-31498 -31371 -31244 -31117 -30990 -30863 -30736 -30609 -30482 -30355
-30228 -30101 -29974 -29847 -29720 -29593 -29466 -29339 -29212 -29085
-28958 -28831 -28704 -28577 -28450 -28323 -28196 -28069 -27942 -27815
-27688 -27561 -27434 -27307 -27180 -27053 -26926 -26799 -26672 -26545
-26418 -26291 -26164 -26037 -25910 -25783 -25656 -25529 -25402 -25275
-25148 -25021 -24894 -24767 -24640 -24513 -24386 -24259 -24132 -24005
-23878 -23751 -23624 -23497 -23370 -23243 -23116 -22989 -22862 -22735
-22608 -22481 -22354 -22227 -22100 -21973 -21846 -21719 -21592 -21465
-21338 -21211 -21084 -20957 -20830 -20703 -20576 -20449 -20322 -20195
-20068 -19941 -19814 -19687 -19560 -19433 -19306 -19179 -19052 -18925
-18798 -18671 -18544 -18417 -18290 -18163 -18036 -17909 -17782 -17655
-17528 -17401 -17274 -17147 -17020 -16893 -16766 -16639 -16512 -16385
-16258 -16131 -16004 -15877 -15750 -15623 -15496 -15369 -15242 -15115
-14988 -14861 -14734 -14607 -14480 -14353 -14226 -14099 -13972 -13845
-13718 -13591 -13464 -13337 -13210 -13083 -12956 -12829 -12702 -12575
-12448 -12321 -12194 -12067 -11940 -11813 -11686 -11559 -11432 -11305
-11178 -11051 -10924 -10797 -10670 -10543 -10416 -10289 -10162 -10035
-9908 -9781 -9654 -9527 -9400 -9273 -9146 -9019 -8892 -8765
-8638 -8511 -8384 -8257 -8130 -8003 -7876 -7749 -7622 -7495
-7368 -7241 -7114 -6987 -6860 -6733 -6606 -6479 -6352 -6225
-6098 -5971 -5844 -5717 -5590 -5463 -5336 -5209 -5082 -4955
-4828 -4701 -4574 -4447 -4320 -4193 -4066 -3939 -3812 -3685
-3558 -3431 -3304 -3177 -3050 -2923 -2796 -2669 -2542 -2415
-2288 -2161 -2034 -1907 -1780 -1653 -1526 -1399 -1272 -1145
-1018 -891 -764 -637 -510 -383 -256 -129 -2 125
252 379 506 633 760 887 1014 1141 1268 1395
1522 1649 1776 1903 2030 2157 2284 2411 2538 2665
2792 2919 3046 3173 3300 3427 3554 3681 3808 3935
4062 4189 4316 4443 4570 4697 4824 4951 5078 5205
5332 5459 5586 5713 5840 5967 6094 6221 6348 6475
6602 6729 6856 6983 7110 7237 7364 7491 7618 7745
7872 7999 8126 8253 8380 8507 8634 8761 8888 9015
9142 9269 9396 9523 9650 9777 9904 10031 10158 10285
10412 10539 10666 10793 10920 11047 11174 11301 11428 11555
11682 11809 11936 12063 12190 12317 12444 12571 12698 12825
12952 13079 13206 13333 13460 13587 13714 13841 13968 14095
14222 14349 14476 14603 14730 14857 14984 15111 15238 15365
15492 15619 15746 15873 16000 16127 16254 16381 16508 16635
16762 16889 17016 17143 17270 17397 17524 17651 17778 17905
18032 18159 18286 18413 18540 18667 18794 18921 19048 19175
19302 19429 19556 19683 19810 19937 20064 20191 20318 20445
20572 20699 20826 20953 21080 21207 21334 21461 21588 21715
21842 21969 22096 22223 22350 22477 22604 22731 22858 22985
23112 23239 23366 23493 23620 23747 23874 24001 24128 24255
24382 24509 24636 24763 24890 25017 25144 25271 25398 25525
25652 25779 25906 26033 26160 26287 26414 26541 26668 26795
26922 27049 27176 27303 27430 27557 27684 27811 27938 28065
28192 28319 28446 28573 28700 28827 28954 29081 29208 29335
29462 29589 29716 29843 29970 30097 30224 30351 30478 30605
30732 30859 30986 31113 31240 31367 31494 31621 31748 31875
32002 32129 32256 32383 32510 32637 32764

